### PR TITLE
4 API onboarding

### PR DESCRIPTION
**Pull request by mere-teresa**
Chapter 4 for a new user guide of our products

### CODE REVIEW

File: doc_sphinx/concepts/cloud.rst [hunk 1]
```diff
@@ -8,25 +8,31 @@ Cloud
...
-A cloud is a global space where doc `projects`_ are hosted. A cloud is a workspace for your company. It also could be a personal playground.
+A cloud is a global space where your `projects`_ are hosted with a similar configuration.
+A cloud contains projects, and each project contains your models, your Document Management System and BCFs.
 
 Cloud administrators are also Projects admin by default, they can see every user in their cloud and change everyone's roles.
 
 Cloud users can't see cloud collaborators. This means that a contractor on a project can't see every collaborator of the company.
 
-Not many actions are available directly on clouds.
+A cloud is a workspace for your company. It also could be a personal playground.
```
Comment (Amoki) [inline doc_sphinx/concepts/cloud.rst]: A cloud can be anything, each use case can use the cloud object differently imho

File: doc_sphinx/tutorials/4_api_onboarding.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.. chapter 4
+
+===============
+API Onboarding
+===============
+
+BIMData API is a tool to interact with your models stored on BIMData's servers.
+Through the API, you can manage your projects, the clouds, upload your IFC files and manage them through endpoints.
```
Comment (Amoki) [inline doc_sphinx/tutorials/4_api_onboarding.rst]: You can also retrieve and update data from models

File: doc_sphinx/tutorials/4_api_onboarding.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+BIMData API follows this general principles:
+
+* All API access is over HTTPS
+* All data is sent and received as JSON
```
Comment (Amoki) [inline doc_sphinx/tutorials/4_api_onboarding.rst]: except for files

File: doc_sphinx/concepts/cloud.rst [hunk 4]
```diff
@@ -8,25 +8,31 @@ Cloud
...
     excerpt
-        Find out more about a Cloud and Cloud users.
+        A cloud is a global space where your projects are hosted.
     endexcerpt
 
 Concept
 ---------
 
-A cloud is a global space where doc `projects`_ are hosted. A cloud is a workspace for your company. It also could be a personal playground.
+A cloud is a global space where your `projects`_ are hosted with a similar configuration.
```
[the remark this answers is not in the document]
Comment (mere-teresa) [reply doc_sphinx/concepts/cloud.rst]: > cloud is bad named

And this name will remain as it is?

File: doc_sphinx/tutorials/4_api_onboarding.rst (hunk 2 above)
Comment (mere-teresa) [reply doc_sphinx/tutorials/4_api_onboarding.rst]: through endpoints => `using the API` is better?

**Comment by mere-teresa** [comment]
En fait, je merge dans la base-branch "new_archi" plutôt que "tech-writing"

**Pull request closed without merging**